site/assets/scss/_content.scss

```diff
@@ -64,6 +64,14 @@
   min-width: 280px;
 }
 
+.table-swatches {
+  // Boosted mod: no `th`
+
+  td code {
+    white-space: nowrap;
+  }
+}
+
 .bd-title {
   @include font-size(3rem);
 }
```

Comment:
Missing .bi change below

Reply:
Added a Boosted mod in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/5eb92ac1345830a824d2404b2800d09d6d036347